docs/src/content/docs/fr/index.mdx

```diff
@@ -39,19 +41,122 @@ import AboutAstro from '~/components/about-astro.astro';
 		Apportez votre langage de balisage préféré. Starlight vous offre une
 		validation frontmatter intégrée avec la sécurité de type TypeScript.
 	</Card>
-	<Card
-		title="Apportez vos propres composants d'interface utilisateur"
-		icon="puzzle"
-	>
+	<Card	title="Apportez vos propres composants d'interface utilisateur" icon="puzzle">
 		Starlight est une solution de documentation complète, indépendante du cadre
 		de travail. Étendez avec React, Vue, Svelte, Solid, et plus encore.
 	</Card>
 </CardGrid>
 
-<AboutAstro title="Présenté par">
-Astro est le framework web tout-en-un conçu pour la vitesse.
-Tirez votre contenu de n'importe où et déployez-le partout, le tout alimenté par vos composants et bibliothèques d'interface utilisateur préférés.
+<TestimonialGrid title="What people are saying">
+	<Testimonial
+		name="Rachel"
+		handle="rachelnabors"
+		cite="https://twitter.com/astrodotbuild/status/1724934718745915558"
+	>
+		L'équipe d'Astro a ÉVOLUÉ la façon dont les documents peuvent être réalisés et vous pouvez tout obtenir dès le départ avec leur projet Starlight.
+	</Testimonial>
+	<Testimonial
+		name="Flavio"
+		handle="flaviocopes"
+		cite="https://twitter.com/flaviocopes/status/1738237658717905108"
+	>
+		Le kit de démarrage officiel d'Astro Starlight est un outil vraiment incroyable pour créer un site web de documentation
+	</Testimonial>
+	<Testimonial
+		name="Tomek"
+		handle="sulco"
+		cite="https://twitter.com/sulco/status/1735610348730802342"
+	>
+		Starlight est notre meilleur exemple d'un bon DX : la rapidité, la commodité et l'attention portée aux détails sont une source d'inspiration.
+		et l'attention portée aux détails sont une source d'inspiration. Il prend soin de la technologie et de l'apparence,
+		pour que vous puissiez vous concentrer sur votre contenu 👏
+
+		L'équipe de StackBlitz l'adore !
+	</Testimonial>
+	<Testimonial
+		name="Roberto"
+		handle="RmeetsH"
+		cite="https://twitter.com/RmeetsH/status/1735783992018760090"
+	>
+		Starlight a changé la donne pour moi, me permettant de me concentrer sur la création de contenu.
+
+		Sa conception intuitive permet non seulement de rationaliser mon flux de travail, mais aussi de réduire le temps d'intégration des développeurs de logiciels libres.
```

Comment:
```suggestion
		Sa conception intuitive permet non seulement de simplifier ma manière de travailler, mais aussi de réduire le temps d'intégration pour les développeurs de logiciels libres.
```

The literal translation sounds a bit weird to me, not sure a lot of people would use "rationaliser" or "flux de travail" in this context.